### ISSUE TITLE
To-Do: Set CI pipeline with github actions

### ISSUE THREAD

**Issue by GiovanniSmokes** [apssouza22/chatflow#14]

**Comment by vporton** [comment]
I see no mean to add an action at https://github.com/apssouza22/chatflow/actions

Apparently, it's so because I have no admin rights.

**Comment by apssouza22** [comment]
@vporton It is done through code. So you would do it to your repo, once it works fine you can open a normal PR

**Comment by vporton** [comment]
@apssouza22 Probably you have an idea which of Python actions to use:
![image](https://github.com/apssouza22/chatflow/assets/2900574/1262df8e-d16f-401a-be52-143fa905d91e)

**Comment by apssouza22** [comment]
@vporton it should be docker related action instead of python. The Required python will be in the docker image

**Comment by vporton** [comment]
See the picture above: There are several docker images for Python.

Maybe, you know which one to choose?

**Comment by apssouza22** [comment]
Test python application might work

**Comment by vporton** [comment]
Python does not provide test mode for an application only to compile and not start.

So, we can test Python only by writing tests (I mean `pytest` or like this). This issue should be postponed to be fulfilled only after we have a testing framework.

**Comment by apssouza22** [comment]
Having the build and the start now would already bring some confidence.
Than we can add integration tests to call the API.

We can start with the integration test as well. Either way is fine.

We want something similar to this
https://dev.to/sahanonp/setup-docker-for-integration-testing-in-github-action-39fn
